lib/import-export-visitor.js

```diff
@@ -338,42 +336,42 @@ module.exports = class ImportExportVisitor extends Visitor {
 
   preserveChild(path, childName) {
     const value = path.getValue();
+    const child = value ? value[childName] : null;
 
-    if (childName) {
-      const child = value ? value[childName] : null;
-
-      if (child && this.code) {
-        this.overwrite(
-          value.start,
-          child.start,
-          ""
-        );
-        this.overwrite(
-          child.end,
-          value.end,
-          ""
-        );
-      }
+    if (child && this.code) {
+      this.overwrite(
+        value.start,
+        child.start,
+        ""
+      );
+      this.overwrite(
+        child.end,
+        value.end,
+        ""
+      );
+    }
 
-      if (this.modifyAST) {
-        // Replace the given path with the child we want to preserve.
-        path.replace(child);
-      }
+    if (this.modifyAST) {
+      // Replace the given path with the child we want to preserve.
+      path.replace(child);
+    }
 
-      path.call(this.visitWithoutReset, childName);
+    path.call(this.visitWithoutReset, childName);
+  }
 
-    } else {
-      if (this.code) {
-        this.overwrite(value.start, value.end, "");
-      }
+  preserveLine(path) {
+    const value = path.getValue();
```

Comment:
They could definitely be private methods. I adopted a convention of prefixing actual AST visitor methods with `visit` so that normal methods could have normal names, but I'm open to more conventions.